### ISSUE TITLE
Little problem with dead/alive state

### ISSUE THREAD

**Issue by pepek112** [rlf/uSkyBlock#1157]
_Please paste the output from `/usb version` below_
```
[09:10:17 INFO]: Name: uSkyBlock
[09:10:17 INFO]: Version: 2.7.5-SNAPSHOT
[09:10:17 INFO]: Description: Ultimate SkyBlock v2.7.6-201809101626-83646d
[09:10:17 INFO]: Language: en (en)
[09:10:17 INFO]:   State: d=128, r=64, i=2, p=2, n=true, awe=false
[09:10:17 INFO]: Server: Paper git-Paper-509 (MC: 1.13.2)
[09:10:17 INFO]:   State: online=true, bungee=false
[09:10:17 INFO]: ------------------------------
[09:10:17 INFO]: Vault 1.7.1-b91 (ENABLED)
[09:10:17 INFO]: WorldEdit 7.0.0-SNAPSHOT;3997-ba8bde0 (ENABLED)
[09:10:17 INFO]: WorldGuard 7.0.0-SNAPSHOT;1784-60731f9 (ENABLED)
[09:10:17 INFO]: Multiverse-Core 2.6.0-b738 (ENABLED)
[09:10:17 INFO]: Multiverse-Portals 2.5.2-SNAPSHOT-b766 (ENABLED)
[09:10:17 INFO]: Multiverse-NetherPortals 2.5.2-SNAPSHOT-b729 (ENABLED)
[09:10:17 INFO]: ------------------------------
```

Hello i have little bug.
After restart island i cant write/send command/teleport player
I think its between dead and alive state 
After restart island, player must relog to do anything,
This applies only to players who are on the reset island
Regards

[Video](https://www.youtube.com/watch?v=rJU-mMokpYo)

**Comment by Ghost-chu** [comment]
我认为删除实体时应判断是否为EntityType.PLAYER

**Comment by Muspah** [comment]
Can you test this again using the latest uSkyBlock-build? Might be fixed in db24030.

Download:
https://oss.pellegrom.pt/snapshots/uSkyBlock/uSkyBlock-20190328-1-ec2a672.jar
